dev-db/clickhouse/clickhouse-9999.ebuild

```diff
@@ -0,0 +1,724 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils llvm systemd toolchain-funcs user
+
+DESCRIPTION="An OSS column-oriented database management system for real-time data analysis"
+HOMEPAGE="https://clickhouse.yandex"
+LICENSE="Apache-2.0"
+
+MY_PN="ClickHouse"
+IUSE="
+	cpu_flags_x86_sse4_1 cpu_flags_x86_sse4_2 cpu_flags_x86_ssse3 cpu_flags_x86_avx
+	cpu_flags_x86_avx2 cpu_flags_x86_popcnt
+	+base64 brotli capnp +client debug doc hdfs3 hyperscan icu +jemalloc json
+	kafka libcxx libedit libressl libunwind llvm lto mongodb mysql odbc parquet
+	protobuf readline +server split-binary ssl static static-libs systemd tcmalloc test tools
+"
+
+REQUIRED_USE="
+	split-binary? ( !static-libs )
+	server? ( cpu_flags_x86_sse4_2 )
+	hdfs3? ( protobuf )
+	static? ( client server tools )
+	libressl? ( ssl )
+	readline? ( client !libedit )
+	libedit? ( client !readline )
+	libcxx? ( !llvm )
+	kafka? ( ssl !arm !x86 !elibc_Darwin !elibc_FreeBSD )
+	?? ( tcmalloc jemalloc )
+	llvm? ( !elibc_Darwin )
+	parquet? ( !elibc_FreeBSD )
+"
+
+if [[ "${PV}" == "9999" ]]; then
+	SRC_URI=""
+	EGIT_REPO_URI="https://github.com/yandex/ClickHouse.git"
+	EGIT_SUBMODULES=( '*'
```

Comment:
Wouldn't it be simple to go for whitelist approach?

Reply:
It is more or less the same, but this way 9999 version will always have all or most features available.

Reply:
@mgorny: How do I disable recursion, e,g. sub-submodules?

Reply:
Probably the same way, i.e. by paths. If it doesn't work, please report a bug.

Reply:
Could you, please, resolve this?